gallery/visualtests/nxTag.js

```diff
@@ -49,25 +52,75 @@ describe('NxTag', function() {
     it('renders a default, Indigo colored NxTag if no color prop is passed', simpleTest(indigoTagSelector));
   });
 
-  describe('NxSelectableTag', function() {
-    const selector = '#nx-selectable-tag-example .nx-tag:first-child',
-        inputSelector = `${selector} input`;
+  describe('unselected NxSelectableTag', function() {
+    function runUnselectedTests(selectableColor) {
+      const selector = `#nx-selectable-tag-example .nx-selectable-color--${selectableColor}`;
+      it('has a lighter background and darker border by default', simpleTest(selector));
+      it('has a dark grey border when hovered', hoverTest(selector));
+      it('has a blue outline when focused', focusTest(selector, undefined, OUTSET));
+      it('has a blue outline and dark grey border when focused and hovered',
+          focusAndHoverTest(selector, undefined, undefined, OUTSET));
+    }
+
+    selectableColors.forEach(color => {
+      describe(color, function() {
+        runUnselectedTests(color);
+      });
+    });
+  });
+
+  describe('selected NxSelectableTag', function() {
+    function runSelectedTests(selectableColor) {
+      const selector = `#nx-selectable-tag-example .nx-selectable-color--${selectableColor}`,
+          inputSelector = `${selector} input`;
+
```

Comment:
If you add a `beforeEach` that clicks the tag you can probably simplify each of the actual tests to use the standard helper functions